README.md

```diff
@@ -36,7 +44,12 @@ Tenés que crear dentro de `cnn/photos` una carpeta por cada categoría que quie
 
 2. Corré el comando:
 ```
-python ./cnn/retrain.py --bottleneck_dir=./cnn/bottleneck --model_dir=./cnn/inception --output_graph=./cnn/retrained_graph.pb --output_labels=./cnn/retrained_labels.txt --image_dir ./cnn/photos
+python ./cnn/retrain.py
+	--bottleneck_dir=./cnn/bottleneck
+	--model_dir=./cnn/inception
+	--output_graph=./cnn/retrained_graph.pb
+	--output_labels=./cnn/retrained_labels.txt
+	--image_dir ./cnn/photos
```

Comment:
I've seen that the = are replaceable with spaces. I'd use only one of the two just to be consistent.